examples/snapshot_json_string_example

```diff
@@ -0,0 +1 @@
+{"application":"application-sample","components":[{"name":"component-sample","containerImage":"quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9","source":{"git":{"url":"git@github.com:redhat-appstudio/integration-service.git\n","revision":"sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c"}}}],"artifacts":{}}
```

Comment:
Does that `git@github.com:redhat-appstudio/integration-service.git\n` need this `\n`?

Reply:
It's created automatically by our code. This is copied from our cluster. So, I think "yes". It's needed.

Reply:
Hmmm, the `\n` is not supposed to be there, we'll probably need to investigate why it's showing up in the build pipeline results...

Reply:
OK.  I removed it.

Reply:
In the code, "snapshotString, _ := json.Marshal(snapshot.Spec)" is used to create the json string. And the snapshot.Spec is like this -
spec:
  application: application-sample
  artifacts: {}
  components:
    - containerImage: >-
        quay.io/redhat-appstudio/sample-image@sha256:34fb1c72bc976937d0a59849160746cec12a8c925f31e74dc9f3b922f74da0e9
      name: component-sample
      source:
        git:
          revision: >-
            sha256:3703843e80f39188c060a6ed0f524519a682be1e7141fee29b878b1973e9783c
          url: |
            git@github.com:redhat-appstudio/integration-service.git
status:  -->  ignore this line, just show the end of the spec

Reply:
Thanks!

Reply:
@jinqi7  `\n` appears again.